python-avocado.spec

```diff
@@ -820,6 +843,32 @@ Pair-Wise algorithms, also known as Combinatorial Independent Testing.
 %{python3_sitelib}/avocado_framework_plugin_varianter_pict*
 %endif
 
+%package -n python2-%{srcname}-plugins-varianter-cit
+Summary: Varianter with combinatorial capabilities by CIT
+Requires: python2-%{srcname} == %{version}
+
+%description -n python2-%{srcname}-plugins-varianter-cit
+This plugin uses a third-party tool to provide variants created by
+Pair-Wise algorithms, also known as Combinatorial Independent Testing.
+
+%files -n python2-%{srcname}-plugins-varianter-cit
+%{python2_sitelib}/avocado_varianter_cit*
+%{python2_sitelib}/avocado_framework_plugin_varianter_cit*
+
+%if %{with_python3}
+%package -n python3-%{srcname}-plugins-varianter-cit
+Summary: Varianter with combinatorial capabilities by CIT
+Requires: python3-%{srcname} == %{version}
+
+%description -n python3-%{srcname}-plugins-varianter-cit
+This plugin uses a third-party tool to provide variants created by
```

Comment:
Well is it really third-party tool? I think it's entirely developed here. How about `A varianter plugin that allows generating variants using "Combinatorial Independent Testing" Pari-Wise algorithm developed at CVUT Prague (https://path/to/research/paper)`.